dotcom-rendering/src/web/components/FixedLargeSlowXIV.tsx

```diff
@@ -16,38 +16,48 @@ export const FixedLargeSlowXIV = ({
 	containerPalette,
 	showAge,
 }: Props) => {
-	const primary = trails[0];
-	const secondary = trails[1];
-	const secondSlice = trails.slice(2, 6);
-	const thirdSlice = trails.slice(6, 14);
+	const firstSlice75 = trails.slice(0, 1);
+	const firstSlice25 = trails.slice(1, 2);
+	const secondSlice25 = trails.slice(2, 6);
+	const thirdSlice25 = trails.slice(6, 14);
```

Comment:
Not sure if we need to rename the second and third slice, but I like the `firstSlice75` and `firstSlice25`

Reply:
I think its still useful to a degree, by looking at the top of the container you can easily tell that the 2nd row is composed of 4 25% cards